.travis.yml

```diff
@@ -23,6 +23,7 @@ env:
     - DOCKER_VERSION=1.12.6 ENABLE_SWARM=1 UPLOAD_SONATYPE=1
     - DOCKER_VERSION=1.13.1 ENABLE_SWARM=1
     - DOCKER_VERSION=17.03.0~ce ENABLE_SWARM=1
+    - DOCKER_VERSION=17.04.0~ce ENABLE_SWARM=1
```

Comment:
assuming Docker is changing the API version for each API change (or that they are not sneaking changes or fixes in) then I think we should test against each API version. Looks to me [like API 1.27 == 17.03.1 and API 1.26 == 17.03.0](https://docs.docker.com/engine/api/v1.27/#section/Versioning), so it sounds like we should either test against both 17.03.0 and 13.03.1 _or_ 13.03.0 and 13.04.0.